test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -195,6 +197,106 @@ public void It_publishes_projects_with_filter_and_rid()
 
 //TODO: Enable testing the run once dotnet host has the notion of looking up shared packages
         }
+
+
+        [Theory]
+        [MemberData(nameof(PublishDocumentationExpectations))]
```

Comment:
Rule of thumb.. 4x3 arguments was more than 2x2.
But you're right, even other tests use InlineData more. I'll change it.